src/configs/git_status.rs

```diff
@@ -0,0 +1,81 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct GitStatusConfig<'a> {
+    pub stashed: SegmentConfig<'a>,
+    pub ahead: SegmentConfig<'a>,
+    pub behind: SegmentConfig<'a>,
+    pub diverged: SegmentConfig<'a>,
+    pub show_sync_count: bool,
+    pub conflicted: SegmentConfig<'a>,
+    pub conflicted_count: CountConfig,
+    pub deleted: SegmentConfig<'a>,
+    pub deleted_count: CountConfig,
+    pub renamed: SegmentConfig<'a>,
+    pub renamed_count: CountConfig,
+    pub modified: SegmentConfig<'a>,
+    pub modified_count: CountConfig,
+    pub staged: SegmentConfig<'a>,
+    pub staged_count: CountConfig,
+    pub untracked: SegmentConfig<'a>,
+    pub untracked_count: CountConfig,
+    pub prefix: &'a str,
+    pub suffix: &'a str,
+    pub style: Style,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for GitStatusConfig<'a> {
+    fn new() -> Self {
+        GitStatusConfig {
+            stashed: SegmentConfig::new("$"),
+            ahead: SegmentConfig::new("⇡"),
+            behind: SegmentConfig::new("⇣"),
+            diverged: SegmentConfig::new("⇕"),
+            conflicted: SegmentConfig::new("="),
+            show_sync_count: false,
+            conflicted_count: CountConfig {
+                enabled: false,
+                style: None,
+            },
```

Comment:
What abount implementing `Default` trait for `CountConfig` and use `CountConfig::default()`?